lib/modules/dosomething/dosomething_api/resources/member_resource.inc

```diff
@@ -90,6 +90,33 @@ function _member_resource_definition() {
       ),
     ),
     'targeted_actions' => array(
+      'update' => [
+        'file' => [
+          'type' => 'inc',
+          'module' => 'dosomething_api',
+          'name' => 'resources/member_resource',
+        ],
+        'help' => 'Update the users profile',
+        'args' => [
+          [
+            'name' => 'id',
+            'optional' => FALSE,
+            'source' => array('path' => 0),
+            'type' => 'string',
+            'description' => 'The Northstar ID of the user we\'re updating.',
```

Comment:
I originally tried doing a generic PUT / Update in the CRUD operations but I wasn't able to get it working properly for some reason, it would only return 404. The targetted action worked so I stuck with that ¯\_(ツ)_/¯

The upside to this way is we could technically let admins now update other users as well if we made some small changes in the permission hook, not that I expect us to actually do that tho